yt_dlp/extractor/rozhlas.py

```diff
@@ -45,3 +47,70 @@ def _real_extract(self, url):
             'duration': duration,
             'vcodec': 'none',
         }
+
+
+class RozhlasVltavaIE(InfoExtractor):
+    _VALID_URL = r'https?://((vltava|wave|radiozurnal|dvojka|plus|sport|d-dur|jazz|junior|pohoda)\.rozhlas|english\.radio)\.cz/[a-z|-]*-(?P<id>\d+)'
```

Comment:
https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#keep-the-regular-expressions-as-simple-as-possible-but-no-simpler

Reply:
done